windows/src/unit-tests/keyboard-info/test-valid.keyboard_info

```diff
@@ -12,9 +12,9 @@
     },
     "license": "mit",
     "languages": [
-        "bal-Arab",
-        "bgn-Arab",
-        "bgp-Arab"
+        "bal",
+        "bgn",
+        "bgp"
```

Comment:
With the improved canonicalization, script should not be present unless it is needed for disambiguation.